### DISCUSSION TITLE
Works with Daikin Altherma 3 R ECH2O (ERGA06EAV3H / EHSXB08P50EF)

### DISCUSSION THREAD

**Discussion by forTT78** [Arnold-n/P1P2MQTT#130] [Show and tell]
Hi, here is my short installation note and big thank you for the project.

My System:
Indoor: EHSXB08P50EF
Outdoor: ERGA06EAV3H

P1P2MQTT directly connected to the P1P2 port via 3 m cable.

Works! :thumbsup:

**Comment by universal-dilettant** [comment]
Just jumping on this discussion: what is the meaning of "works!"? Works as in reading parameters or works as in writing commands?
I have the same Daikin HW as mentioned before, had acquired the P1P2mqtt box from Arnold in Jan this year and installed some days ago. After upgrading to the latest FW for both ESP and atmega, the box shows up in mqtt and can be accessed via telnet alright. However, and this is maybe a dumb question: I don't succeed to send commands like i.e. "Heating on" (= E 35002F01 according to the documentation for E-series) via telnet. Any help is appreciated.

![grafik](https://github.com/user-attachments/assets/008e1387-60d0-4cbc-9314-d839be386271)

**Reply by Arnold-n** [reply]
Your bridge is not yet operating in control mode. If you enter "L1" it should start control mode, and you can enter commands like "E35002F01".

**Reply by forTT78** [reply]
Yes, "heating on" and controlling works, e.g. like adjusting the LWT temperature. I use home assistant for it.

**Comment by universal-dilettant** [comment]
Works! At least, to start the heat pump alright. Sometimes, you can't see the wood for the trees. Thank you for the hint.

However, next question. Is it normal to get those messages after enabling command mode, like:
  _paramNr 0x017C > expected nr_params 380 for packetType 0x35
  paramNr 0x00AF > expected nr_params 175 for packetType 0x3B_

![image](https://github.com/user-attachments/assets/42c7b967-9c80-4584-b29a-baf96f1b4ad0)

**Comment by Arnold-n** [comment]
Daikin continues to increase the number of parameters reported in newer models, which causes these error messages.  [E-series firmware v0.9.54rc4](https://github.com/Arnold-n/P1P2MQTT/blob/main/Firmware_images/P1P2MQTT-bridge-v0.9.54rc4-Daikin-E.ino.bin) should resolve this error message.

**Reply by forTT78** [reply]
Only flashing of the new bridge firmware necessary, right?
Directly via the web portal.

**Reply by Arnold-n** [reply]
Indeed!

**Reply by forTT78** [reply]
Thank you.  Up and running:
```
[ESP] 2024-10-11 15:26:12 P1P2MQTT bridge v0.9.54rc4
[ESP] 2024-10-11 15:26:12 Compiled Oct 11 2024 13:30:30 for Daikin E-Series
```

**Comment by universal-dilettant** [comment]
Flashed alright.

If I may (mis-?) use this thread again: what is the meaning of 
_Throttling at 70 ... 1?_

![grafik](https://github.com/user-attachments/assets/b2eaa348-d764-4a7f-bdbe-9fdedaccd5da)

**Comment by Arnold-n** [comment]
It throttles outgoing MQTT traffic during the first 100s after boot/reconnection/D3/... to prevent MQTT output buffer overruns.
